Documentation/Routing/SimplifyFacetsInQuery.rst

```diff
@@ -0,0 +1,151 @@
+.. _routing-simplify-facets:
+
+===================================
+Simplify facets inside of the query
+===================================
+
+Sometimes you or your customer want to share links to a search result. The link itself could really hard to read, a specially if it contains a lot of filters.
+
+There are two options that you can apply on the query string:
+
+1. Concat multiple values of facet into a single string
+2. Mask the filter inside of the query
+
+All query related configurations will be places below node `query` below node `solr`:
+
+.. code-block:: yaml
+
+  routeEnhancers:
+    products:
+      solr:
+        query:
+
+Concat multiple values into a string
+------------------------------------
+
+In order to active concat of multiple value add entry `concat` and set the value to `true`.
+
+By default the values will concat with a colon. If you need a different separator, add the entry `valueSeparator` and set a value that fits your need.
+
+.. code-block:: yaml
+
+  routeEnhancers:
+    products:
+      solr:
+        query:
+          concat: true
+          # valueSeparator: ','
+
+Before Solr concat the values, a alphanumeric sort applied.
+
+Imaging following filter:
+
+	tx_solr[filter][0]=taste:sweet&tx_solr[filter][1]=taste:sour&tx_solr[filter][2]=taste:matcha
+
+By enable concat of multiple values, it would change into
+
+	tx_solr[filter][0]=taste:matcha,sour,sweet
+
+
+Mask filter inside of the query
+-------------------------------
+
+Mask filter inside of the URL allows you to make the URL more readable and to shrink the length.
+
+The configuration contains two parts:
+
+1. enable the mask
+2. map from facet to parameter
```

Comment:
Here we need exact term or may be short explanation(could be in brackets or hint) to "facet".
May be the point/parallels to the TypoScript configs.

Reply:
The sentence changed into `map from the facet name to parameter`.
As there is no glossary available, a link to the TypoScript description placed behind the sentence.